views/scss/proctoring.scss

```diff
@@ -77,7 +77,7 @@
         }
     }
     
-    .diagnostic-index, .reporting-index, .delivery-monitoring {
+    .diagnostic-index, .reporting-index, .delivery-monitoring, .delivery-manager {
```

Comment:
I would have the content width of .delivery-manager to be like the one for .delivery-testtakers (=800px) to show to the user that those two screns are related and different from the monitoring one. Plus, I would add event more visual clues to .delivery-manager and  .delivery-testtakers content, e.g. an additional border, or background. It does need to be too complex but just adding small visual differences would prevent user from being confused I think.